sound/sound_library.h

```diff
@@ -115,20 +115,44 @@ class SoundLibrary {
         if (hundredths != 0) Play(SoundToPlay(&SFX_mnum, hundredths - 1));
         break;
       case SAY_WHOLE:
+        if (number == 0) {
+          Play("mzero.wav");
+          break;
+        }
+#ifdef BLADE_ID_SCAN_MILLIS
+        // Billions spoken as "thousand million"
+        if (billion > 0) {
+          TensValue(billion);
+          Play("thousand.wav");
+          Play("million.wav");
+        }
+        // Millions
+        if (million > 0) {
+          TensValue(million);
+          Play("million.wav");
+        }
+#endif
         // Thousands
+        if (thousand >= 100) {
+          /* Support hundreds thousands
```

Comment:
I do not know where this version of Thousand came from, but it's in my local copy.
It works fine and I am only testing the official version as we speak.
Does it look like a good change? Any idea where this came from somewhere along the way?